CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#10](https://github.com/green-code-initiative/ecoCode-php/pull/10) Adding EC35 rule : EC35 rule replaces EC34 with a specific use case ("file not found" specific)
 - [#13](https://github.com/green-code-initiative/ecoCode-php/issues/13) Add build number to manifest
 - Update ecocode-rules-specifications to 0.0.9
+- Fix unit tests execution with Maven
```

Comment:
please put the issue number + a link to it

Reply:
Oops sorry, done!